example/tests/unit/test_renderers.py

```diff
@@ -137,3 +139,14 @@ class EmptyRelationshipViewSet(views.ReadOnlyModelViewSet):
     assert 'relationships' in result['data']
     assert 'bio' in result['data']['relationships']
     assert result['data']['relationships']['bio'] == {'data': None}
+
+
+@pytest.mark.django_db
+def test_extract_relation_instance(comment_factory):
```

Comment:
pytest-factory creates one instance per registered factory. So you can simply use `comment` fixture here and delete the next line.

Reply:
done